utils/constants.tsx

```diff
@@ -450,6 +450,11 @@ export const ModalIdentifiers = {
     WORK_TEMPLATES: 'work_template',
     DOWNGRADE_MODAL: 'downgrade_modal',
     PURCHASE_IN_PROGRESS: 'purchase_in_progress',
+    DELETE_WORKSPACE: 'delete_workspace',
+    FEEDBACK: 'feedback',
+    DELETE_WORKSPACE_PROGRESS: 'delete_workspace_progress',
+    DELETE_WORKSPACE_SUCCESS: 'delete_workspace_success',
+    DELETE_WORKSPACE_FAILURE: 'delete_workspace_failure',
```

Comment:
Can we consolidate these more? It feels awkward to me to close and open different modals for what is the same experience.

Using the SuccessModal for downgrade as its own separate modal makes sense to me, though.